components/resources/commands.grdp

```diff
@@ -0,0 +1,487 @@
+    Bookmark Bar Rename Folder
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_EDIT">
+    Bookmark Bar Edit
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_REMOVE">
+    Bookmark Bar Remove
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_UNDO">
+    Bookmark Bar Undo
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_REDO">
+    Bookmark Bar Redo
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_ADD_NEW_BOOKMARK">
+    Bookmark Bar Add New Bookmark
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_NEW_FOLDER">
+    Bookmark Bar New Folder
+  </message>
+  <message name="IDS_IDC_BOOKMARK_MANAGER">
+    Bookmark Manager
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_ALWAYS_SHOW">
+    Bookmark Bar Always Show
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_SHOW_APPS_SHORTCUT">
+    Bookmark Bar Show Apps Shortcut
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_SHOW_READING_LIST">
+    Bookmark Bar Show Reading List
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_SHOW_MANAGED_BOOKMARKS">
+    Bookmark Bar Show Managed Bookmarks
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_ADD_TO_BOOKMARKS_BAR">
+    Bookmark Bar Add To Bookmarks Bar
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_REMOVE_FROM_BOOKMARKS_BAR">
+    Bookmark Bar Remove From Bookmarks Bar
+  </message>
+  <message name="IDS_IDC_CONTENT_CLIPBOARD_HISTORY_MENU">
+    Content Clipboard History Menu
+  </message>
+  <message name="IDS_IDC_MEDIA_ROUTER_SHOW_IN_TOOLBAR">
+    Show Media Router In Toolbar
+  </message>
+  <message name="IDS_IDC_MEDIA_STREAM_DEVICE_STATUS_TRAY">
+    Media Stream Device Status Tray
+  </message>
+  <message name="IDS_IDC_MEDIA_STREAM_DEVICE_ALWAYS_ALLOW">
+    Media Stream Device Always Allow
+  </message>
+  <message name="IDS_IDC_TAB_SEARCH">
+    Tab Search
+  </message>
+  <message name="IDS_IDC_TAB_SEARCH_CLOSE">
+    Tab Search Close
+  </message>
+  <message name="IDS_IDC_DEBUG_TOGGLE_TABLET_MODE">
+    Debug Toggle Tablet Mode
+  </message>
+  <message name="IDS_IDC_DEBUG_PRINT_VIEW_TREE">
+    Debug Print View Tree
+  </message>
+  <message name="IDS_IDC_DEBUG_PRINT_VIEW_TREE_DETAILS">
+    Debug Print View Tree Details
+  </message>
+  <message name="IDS_IDC_LIVE_CAPTION">
+    Toggle Live Captions
+  </message>
+  <message name="IDS_IDC_RUN_SCREEN_AI_VISUAL_ANNOTATIONS">
+    Run screen AI visual annotations
+  </message>
+  <message name="IDS_IDC_SIDEBAR_SHOW_OPTION_ALWAYS">
+    Show Sidebar
+  </message>
+  <message name="IDS_IDC_SIDEBAR_SHOW_OPTION_MOUSEOVER">
+    Show Sidebar on Mouseover
+  </message>
+  <message name="IDS_IDC_SIDEBAR_SHOW_OPTION_NEVER">
+    Hide Sidebar
+  </message>
+  <message name="IDS_IDC_SHOW_BRAVE_WALLET_PANEL">
+    Show Brave Wallet Panel
+  </message>
+  <message name="IDS_IDC_CLOSE_BRAVE_WALLET_PANEL">
+    Close Brave Wallet Panel
+  </message>
+  <message name="IDS_IDC_SHOW_BRAVE_VPN_PANEL">
+    Show Brave Vpn Panel
```

Comment:
All the usages for `Vpn` we'll want to change to be `VPN` which is how we stylize it in the rest of the product